sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestHelper.scala

```diff
@@ -57,7 +57,7 @@ trait SQLQueryTestHelper extends SQLConfHelper with Logging {
       .replaceAll("Created By.*", s"Created By $notIncludedMsg")
       .replaceAll("Created Time.*", s"Created Time $notIncludedMsg")
       .replaceAll("Last Access.*", s"Last Access $notIncludedMsg")
-      .replaceAll("Owner\t.*", s"Owner\t$notIncludedMsg")
+      .replaceAll("Owner.*", s"Owner $notIncludedMsg")
```

Comment:
Isn't it too general because `Owner` is too short?

BTW, some test cases seem to fail.

```
[info] - describe.sql *** FAILED *** (422 milliseconds)
[info]   describe.sql
[info]   Expected "...me	test_schema
[info]   Owner[	][not included in com...", but got "...me	test_schema
[info]   Owner[ ][not included in com..." Result did not match for query #28
```